src/backend/utils/fmgr/fmgr.c

```diff
@@ -47,7 +48,8 @@ PGDLLIMPORT fmgr_hook_type fmgr_hook = NULL;
 PGDLLIMPORT non_tsql_proc_entry_hook_type non_tsql_proc_entry_hook = NULL;
 PGDLLIMPORT get_func_language_oids_hook_type get_func_language_oids_hook = NULL;
 PGDLLIMPORT pgstat_function_wrapper_hook_type pgstat_function_wrapper_hook = NULL;
-set_local_schema_for_func_hook_type set_local_schema_for_func_hook;
+pltsql_pgstat_function_check_hook_type pltsql_pgstat_function_check_hook = NULL;
+set_local_schema_for_func_hook_type set_local_schema_for_func_hook = NULL;
```

Comment:
why this change?

Reply:
unrelated change. Just for consistency in the file.